creator-node/src/dbManager.js

```diff
@@ -203,6 +203,48 @@ class DBManager {
       log(`completed in ${Date.now() - start}ms`)
     }
   }
+
+  /**
+   * Retrieves md5 hash of all File multihashes for user ordered by clock asc, optionally by clock range
+   *
+   * @param {Object} lookupKey lookup user by either cnodeUserUUID or walletPublicKey
+   * @param {Number?} clockMin if provided, consider only Files with clock >= clockMin (inclusive)
+   * @param {Number?} clockMax if provided, consider only Files with clock < clockMax (exclusive)
+   * @returns {Number} filesHash
+   */
+  static async fetchFilesHashFromDB({
+    lookupKey: { lookupCNodeUserUUID, lookupWallet },
+    clockMin = null,
+    clockMax = null
+  }) {
+    let subquery = 'select multihash from "Files"'
+
+    if (lookupWallet) {
+      subquery += ` where "cnodeUserUUID" = (
+        select "cnodeUserUUID" from "CNodeUsers" where "walletPublicKey" = '${lookupWallet}'
+      )`
+    } else {
+      subquery += ` where "cnodeUserUUID" = '${lookupCNodeUserUUID}'`
+    }
+
+    if (clockMin) {
+      // inclusive
+      subquery += ` and clock >= ${parseInt(clockMin)}`
```

Comment:
we need to escape these values or it could make us vuln to injection style attacks. lets use something like replacements to inject these values https://sequelize.org/master/manual/raw-queries.html#replacements

Reply:
good call, i meant to but forgot - done